src/framework.js

```diff
@@ -0,0 +1,19 @@
+import DashboardController from './controllers/dashboard.controller';
+
+class Framework {
+  /**
+   * Select the route handler and call App#render.
```

Comment:
**Kindly remove commented code, you can always use git history to access previous code right :slightly_smiling_face: **